client/signup/steps/domains/index.jsx

```diff
@@ -487,7 +487,7 @@ class DomainsStep extends React.Component {
 				isEligibleVariantForDomainTest={ this.isEligibleVariantForDomainTest() }
 				suggestion={ initialQuery }
 				designType={ this.getDesignType() }
-				vendor={ getSuggestionsVendor( true ) }
+				vendor={ getSuggestionsVendor( { isSignup: ! this.props.isDomainOnly } ) }
```

Comment:
Done in https://github.com/Automattic/wp-calypso/pull/44869/files#diff-dac1653ca8a4fe77505424b91353f493df971ad43841fce152c3572a13738a65